config/environments/test.rb

```diff
@@ -39,4 +39,20 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  # Setup mailer
+
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.default charset: "utf-8"
+
+  ActionMailer::Base.smtp_settings = {
+    address: "smtp.gmail.com",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.